packages/venia-concept/src/components/Footer/footerTile.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from 'src/components/Icon';
+import classify from 'src/classify';
+import defaultClasses from './footerTile.css';
+import { footerTilePropType } from './constants';
+
+const iconAttrs = {
+    width: 18,
+    color: 'rgb(var(--venia-black))'
+};
+
+class FooterTile extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            root: PropTypes.string,
+            body: PropTypes.string,
+            title: PropTypes.string,
+            iconContainer: PropTypes.string
+        }),
+        item: footerTilePropType
+    };
+
+    getHeaderTitle = () => {
+        const {
+            item: { headerTitle }
+        } = this.props;
+        const typeofHeaderTitle = typeof headerTitle;
+
+        if (typeofHeaderTitle === 'function') {
+            return headerTitle();
+        } else if (typeofHeaderTitle === 'string') {
+            return headerTitle;
+        } else {
+            throw new Error('Incorrect type of header title');
+        }
+    };
+
+    render() {
+        const {
+            classes,
+            item: { iconName, bodyText }
+        } = this.props;
+
+        return (
+            <div className={classes.root}>
+                <h2 className={classes.title}>
+                    <span className={classes.iconContainer}>
+                        <Icon name={iconName} attrs={iconAttrs} />
+                    </span>
+                    <span>{this.getHeaderTitle()}</span>
```

Comment:
It will be better to use div here. It is incorrect to render block element inside inline element, here it is possible to to this when headerTitle function return block element.